shotover-proxy/benches/benches/cassandra.rs

```diff
@@ -42,13 +33,16 @@ fn cassandra(c: &mut Criterion) {
         });
         for query in &queries {
```

Comment:
Can we get the list of queries from something like: `resources.queries.iter_keys()`?

Reply:
Did try that but it's a function until it gets passed to `bench_with_input`

Reply:
Oh true, because we need the query type to construct the bench name really early.
Quite unfortunate but I cant think of a better approach.